src/vaadin-avatar.js

```diff
@@ -117,6 +117,10 @@ class AvatarElement extends ElementMixin(ThemableMixin(PolymerElement)) {
           border-radius: 50%;
           box-shadow: inset 0 0 0 2px var(--vaadin-avatar-user-color);
         }
+
+        :host([clickable]) {
+          cursor: pointer;
```

Comment:
https://vaadin.com/docs/latest/ds/foundation/interaction/#clickable-cursor

```suggestion
          cursor: var(--lumo-clickable-cursor);
```

This should be set in the Lumo theme of the component. I think the default could still be `pointer` here in the `src` version. The Material theme can inherit the same default.

Reply:
@jouni, `--lumo-clickable-cursor` resolves to `default`, so it should not have any kind of pointer in Lumo?

Reply:
Yeah, then it’s consistent with other clickable elements. Links are an exception, they use the browser default.

Reply:
In addition to a cursor, we could use a subtle hover effect for clickable avatars. Not sure if we still have a before/after pseudo element available for use, but we could overlay the avatar with `--lumo-shade-10pct`, for example, on hover.

Reply:
I pushed updated styles for Lumo now.

Reply:
I’ll do something for Material as well.

Reply:
Done. Though, now this starts to beg the question that we should implement the `[active]` state as well when you use the Enter or Space keys to click the avatar. Now you don’t get the same visual feedback about the click as you do with the mouse.